initial-commit.sh

```diff
@@ -22,6 +22,12 @@ case "$REPO_NAME" in
     ;;
 esac
 
+case "$REPO_NAME" in
+    *-docker)
```

Comment:
what's this line doing?

Reply:
checks the repo name against the string for replacing.  See same functionality above for the Terraform repo template.

Reply:
ahh, it's the suffix of the template repo name. $REPO_NAME should probably be $TEMPLATE_REPO_NAME